### PR TITLE
add file line count info on UI

### LINKED ISSUE #8170: Show line counts in file header
The file header currently only shows file size:

Screen Shot 2019-09-13 at 10 35 48

It would be useful if it would also show the number of lines and possibly SLOC, like GitHub:

Screen Shot 2019-09-13 at 10 35 33

### PR DESCRIPTION
**Pull request by silverwind**
Also reworked the header to remove the filename (which is redundant with the file path above) and made the header a flexbox with a monospace font.

Fixes: https://github.com/go-gitea/gitea/issues/8170

### CODE REVIEW

**Review by lafriks — APPROVED** [review]

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -681,6 +681,7 @@ stored_lfs = Stored with Git LFS
 commit_graph = Commit Graph
 blame = Blame
 normal_view = Normal View
+lines = lines
```
Comment (guillep2k) [inline options/locale/locale_en-US.ini:684]: Not that I mind, but this doesn't cover the "1 line" case. Otherwise, LG-TM.

**Review by guillep2k — APPROVED** [review]

**Comment by lunny** [comment]
I think we still need to follow https://github.com/go-gitea/gitea/pull/8396#discussion_r331808468

File: options/locale/locale_en-US.ini (hunk 1 above)
Comment (silverwind) [reply options/locale/locale_en-US.ini:684]: Indeed, will probably do a followup on that.

Comment (lafriks) [reply options/locale/locale_en-US.ini:684]: @silverwind I already created PR for that (already merged ;) )

**Pull request merged**